app/i18n/en/index.ts

```diff
@@ -2335,6 +2343,15 @@ const en: BaseTranslation = {
     }
   },
   AccountScreen: {
+    fundsMoreThan5Dollars: "Your account has more than $5",
```

Comment:
amount should be localized eventually in the local currency used (maybe it should be done as part of another PR)